java/ingest/ingest-runner/src/test/java/sleeper/ingest/testutils/ResultVerifier.java

```diff
@@ -105,9 +105,10 @@ public static List<Record> readMergedRecordsFromPartitionDataFiles(Schema sleepe
                                                                        List<FileReference> fileReferenceList,
                                                                        Configuration hadoopConfiguration) {
         List<Record> recordsRead = new ArrayList<>();
-        for (FileReference fileReference : fileReferenceList) {
+        Set<String> filenames = fileReferenceList.stream().map(FileReference::getFilename).collect(Collectors.toSet());
+        for (String filename : filenames) {
```

Comment:
Does this not cause a problem for assertions, since the order of the files will change?

Reply:
Thanks, i've reverted to using the list and keeping track of duplicates in a set